.github/workflows/publish_release_draft.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish draft release when release branch is merged
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+      - support/*
+
+jobs:
+  publish_release:
+    if: ${{ github.event.pull_request.merged }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get release id
+        id: get_release_id
+        run: |
+          TOKEN=${{ secrets.GITHUB_TOKEN }}
+          RELEASEID=$(curl -H "Accept: application/vnd.github+json" -H "Authorization: token $TOKEN" https://api.github.com/repos/werkbot/silverstripe-spam-module/releases)
```

Comment:
This is a raw curl call to the github actions api. You will need to update the `https://api.github.com/repos/werkbot/silverstripe-spam-module/releases` to your fork.